xmlWorksheet.go

```diff
@@ -231,11 +231,18 @@ type xlsxRow struct {
 // currently I have not checked it for completeness - it does as much
 // as I need.
 type xlsxC struct {
-	R string `xml:"r,attr"`           // Cell ID, e.g. A1
-	S int    `xml:"s,attr"`           // Style reference.
-	T string `xml:"t,attr,omitempty"` // Type.
-	V string `xml:"v"`                // Value
-	F *xlsxF `xml:"f,omitempty"`      // Formula
+	R  string  `xml:"r,attr"`           // Cell ID, e.g. A1
+	S  int     `xml:"s,attr"`           // Style reference.
+	T  string  `xml:"t,attr,omitempty"` // Type.
+	V  string  `xml:"v"`                // Value
+	F  *xlsxF  `xml:"f,omitempty"`      // Formula
+	IS *xlsxIS `xml:"is"`               // Inline string
```

Comment:
Yup, good point! Iirc write support is rudimentary, but existent. 

Sent from my iPhone

> On Sep 7, 2016, at 9:57 AM, Gareth Watts <notifications@github.com> wrote:
> 
> In xmlWorksheet.go:
> 
> > @@ -231,11 +231,18 @@ type xlsxRow struct {
> >  // currently I have not checked it for completeness - it does as much
> >  // as I need.
> >  type xlsxC struct {
> > -	R string `xml:"r,attr"`           // Cell ID, e.g. A1
> > -	S int    `xml:"s,attr"`           // Style reference.
> > -	T string `xml:"t,attr,omitempty"` // Type.
> > -	V string `xml:"v"`                // Value
> > -	F *xlsxF `xml:"f,omitempty"`      // Formula
> > +	R  string  `xml:"r,attr"`           // Cell ID, e.g. A1
> > +	S  int     `xml:"s,attr"`           // Style reference.
> > +	T  string  `xml:"t,attr,omitempty"` // Type.
> > +	V  string  `xml:"v"`                // Value
> > +	F  *xlsxF  `xml:"f,omitempty"`      // Formula
> > +	IS *xlsxIS `xml:"is"`               // Inline string
> not sure when this struct is ever encoded, but should IS also have ,omitempty set if it's not commonly used?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>